examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -599,6 +615,27 @@ def remove_special_characters(batch):
     # derive max & min input length for sample rate & max duration
     max_input_length = data_args.max_duration_in_seconds * feature_extractor.sampling_rate
     min_input_length = data_args.min_duration_in_seconds * feature_extractor.sampling_rate
+
+    if data_args.max_eval_duration_in_seconds is None:
+        logger.warning(
```

Comment:
For deprecation warnings we usually use `warnings.warn("Text", FutureWarning)` as shown here: https://github.com/huggingface/transformers/blob/957ce6465aa7f2cf89cbfd37d6cf166dd588ceab/src/transformers/models/wav2vec2/modeling_wav2vec2.py#L1216